src/components/ContextualMenu/ContextualMenuDropdown/ContextualMenuDropdown.tsx

```diff
@@ -243,27 +243,19 @@ const ContextualMenuDropdown = <L,>({
       bottom: toggleRect.bottom - scrollableParentRect.top,
     };
 
-    // Calculate the rect in relation to the Window
-    const windowRect = {
```

Comment:
For context, this has been removed as the only attribute that is later referenced, is windowRect.height, i.e; window.innerHeight - as replaced/referenced in line 252.